deltachat-ios/Controller/WebxdcViewController.swift

```diff
@@ -62,24 +62,37 @@ class WebxdcViewController: WebViewViewController {
         
         let script = """
         window.webxdc = (() => {
-          let setUpdateListenerPromise = null
           var log = (s)=>webkit.messageHandlers.log.postMessage(s);
         
           var update_listener = () => {};
 
-          window.__webxdcUpdate = async (lastSerial) => {
+          let should_run_again = false;
+          let running = false;
+          let lastSerial = 0;
+          window.__webxdcUpdate = async () => {
+            if (running) {
+                should_run_again = true
+                return
+            }
+            if (should_run_again) {
+                should_run_again = false
+            }
+            running = true;
             try {
-                const updates = await fetch("webxdc-update.json?"+lastSerial).then((response) => response.json())
+                var updates = await fetch("webxdc-update.json?"+lastSerial).then((response) => response.json())
```

Comment:
why is `const` no longer correct?

Reply:
const is correct, but apple gave me an unhelpful message about the await keyword (I forgot the async keyword above and it did not tell me that instead it complained about the fetch keyword, so I was confused and tried all kinds of things out, switching to var was one of those things

Reply:
```suggestion
                const updates = await fetch("webxdc-update.json?"+lastSerial).then((response) => response.json())
```